### DISCUSSION TITLE
DeckGL compatibility with Safari 15?

### DISCUSSION THREAD

**Discussion by mzanguyenbunnings** [visgl/deck.gl#6126] [Q&A]
Hi all,

Looks like DeckGL latest release is not compatible with Safari 15, which is going to be shipped with ios15 and macOS12. Do we have any plan to support Safari 15 in the pipeline?

Below is what deckGL's sample page rendered in Safari 15 beta
![image](https://user-images.githubusercontent.com/80663910/130548714-ac90050c-c115-43cd-bc7c-60059f47e0c0.png)

**Comment by Pessimistress** [comment]
@ibgreen I see this after turning on logging:

```
Error linking: Internal error compiling shader with Metal backend.
Please submit this shader, or website as a bug to https://bugs.webkit.org
```

Edit: all examples on https://luma.gl seem to be working as expected

Edit: Safari 15 fails to compile shaders containing `0.0/0.0`. Work around: https://github.com/visgl/luma.gl/pull/1491

**Reply by Pessimistress** [reply]
It is working now with v8.5.5.

**Reply by mzanguyenbunnings** [reply]
Thanks @Pessimistress , confirming that it's now working
